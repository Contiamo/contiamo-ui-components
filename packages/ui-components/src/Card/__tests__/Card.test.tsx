import * as React from "react"
import { shallow } from "enzyme"

import ThemelessCard from "../Card"
import wrapDefaultTheme from "../../../utils/wrap-default-theme"

const Card = wrapDefaultTheme(ThemelessCard)

describe("Card", () => {
  it("Should render", () => {
    const renderedComponent = shallow(<Card>hi</Card>)
    expect(renderedComponent).toMatchSnapshot()
  })
})
