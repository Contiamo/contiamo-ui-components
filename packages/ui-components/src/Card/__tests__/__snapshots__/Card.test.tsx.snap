// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Should render 1`] = `
<ThemeProvider
  theme={
    Object {
      "baseZIndex": 0,
      "colors": Object {
        "black": "#000000",
        "error": "#DE1A1A",
        "grey10": "#F5F5F5",
        "grey20": "#F1F1F1",
        "grey30": "#D0D9E5",
        "grey40": "#C6D1E1",
        "grey50": "#BBCADC",
        "grey60": "#999999",
        "grey70": "#8092B0",
        "grey80": "#747474",
        "grey90": "#445873",
        "info": "#1499CE",
        "success": "#689F2C",
        "warning": "#FFAE00",
        "white": "#FFFFFF",
      },
      "spacing": 12,
      "typography": Object {
        "body": Object {
          "fontSize": 13,
          "fontWeight": 400,
          "letterSpacing": "normal",
          "lineHeight": "1.5",
          "opacity": 1,
          "textTransform": "none",
        },
        "heading1": Object {
          "fontSize": 13,
          "fontWeight": 700,
          "letterSpacing": "normal",
          "lineHeight": "1.5",
          "opacity": 1,
          "textTransform": "none",
        },
        "heading2": Object {
          "fontSize": 13,
          "fontWeight": 700,
          "letterSpacing": "normal",
          "lineHeight": "1.5",
          "opacity": 0.6,
          "textTransform": "none",
        },
        "small": Object {
          "fontSize": 12,
          "fontWeight": 400,
          "letterSpacing": "normal",
          "lineHeight": "1.5",
          "opacity": 1,
          "textTransform": "none",
        },
        "title": Object {
          "fontSize": 22,
          "fontWeight": 600,
          "letterSpacing": "normal",
          "lineHeight": "1.5",
          "opacity": 1,
          "textTransform": "none",
        },
      },
    }
  }
>
  <glamorous(Card)>
    hi
  </glamorous(Card)>
</ThemeProvider>
`;
